recipes/desdeo/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "desdeo" %}
+{% set version = "0.1.3" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "1f06b9cfecde8e14332b12b2bd88d740ecd3765d00462a4b8dc9e923c46dcb2c" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - pip
+    - python >=3.6
+    - pytest-runner
+  run:
+    - python >=3.6
+    - numpy
+    - prompt_toolkit
+    - scikit-learn
+    - scipy
+
+test:
+  imports:
+    - desdeo
```

Comment:
Because importing `desdeo` does not appear to import any submodules, please add:

```
    - desdeo.core
    - desdeo.method
   ... etc
```